legal-api/src/legal_api/reports/business_document.py

```diff
@@ -550,6 +550,12 @@ def _set_continuation_in_details(self, business: dict):
                     'business_id': jurisdiction.business_id,
                     'filing_id': jurisdiction.filing_id,
                     }
+            
+            # Imported from COLIN
+            if continuation_in_filing.source == Filing.Source.COLIN.value:
```

Comment:
Think using `is_tombstone` to check will be better. [Reference](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/business_document.py#L332)